### DISCUSSION TITLE
调节音量时，显示控制窗口太大了，而且显示时长太长了，能不能设置显示大小和时间啊？

### DISCUSSION THREAD

**Discussion by qionglaoer** [shuax/MouseInc.Settings#323] [Q&A] [answered]
用鼠标滚轮在屏幕边缘调节音量时，显示控制窗口太大了，而且显示时长太长了，能不能设置显示大小和时间啊？其实只显示音量条就可以了，不想显示控制当前声音程序。

**Comment by Ctrl-Ling** [comment, marked as answer]
这个是Windows自带的功能

**Comment by 6Tom** [comment]
这实在是。。。这只是调用windows自带的接口，你提这个需要自己另行开发ui界面，麻烦炸了

**Comment by 6Tom** [comment]
时隔多日，因为找到了一些解决方法，回来分享给你

有俩小软件，都可以结合自定义键位+鼠标滚轮调节音量，可以自定义滚动条显示的样式、位置、滞留时间，应该可以满足你这需求

其中volumouse有个小问题，当焦点在uwp应用上时无法使用

volume2是开源软件，也有个小问题，把音量控制方式设为对数或平面的时候，滚动条显示的方式和系统音量不一致
我自用的是volume2，它的功能更多更完善一些，还集成了管理员自启任务，还有按时段自动调整音量的一些玩法

链接供参考
[http://www.nirsoft.net/utils/volumouse.html](url)
[https://github.com/irzyxa/Volume2](url)

**Reply by qionglaoer** [reply]
特别感谢你的回复！
